pyiron_base/project/archiving/export_archive.py

```diff
@@ -46,18 +60,20 @@ def copy_files_to_archive(
     directory_to_transfer, archive_directory, compressed=True, copy_all_files=False
 ):
     """
-    Create an archive of jobs in directory_to_transfer.
+    Copy files from a directory to an archive, optionally compressing the archive.
 
     Args:
-        directory_to_transfer (str): project directory with jobs to export
-        archive_directory (str): name of the final archive; if no file ending is given .tar.gz is added automatically when needed
-        compressed (bool): if True compress archive_directory as a tarball; default True
-        copy_all_files (bool): if True include job output files in archive, otherwise just include .h5 files; default False
-    """
+        directory_to_transfer (str): The directory containing the files to transfer.
+        archive_directory (str): The destination directory for the archive.
+        compressed (bool): If True, compress the archive directory into a tarball. Default is True.
+        copy_all_files (bool): If True, include all files in the archive, otherwise only .h5 files. Default is False.
 
+    """
     assert isinstance(archive_directory, str) and ".tar.gz" not in archive_directory
-    # print("directory to transfer: "+directory_to_transfer)
-    dst = os.path.join(archive_directory, getdir(path=directory_to_transfer))
+    dir_name_transfer = os.path.basename(directory_to_transfer) or os.path.basename(
+        os.path.dirname(directory_to_transfer)
+    )
```

Comment:
```suggestion
    dir_name_transfer = getdir(path=directory_to_transfer)
```
Seems like another `getdir` snuck in here, but otherwise lgtm.